kernelci/config/base.py

```diff
@@ -24,15 +24,64 @@
 # Common classes for all config types
 #
 
-class YAMLObject:
+class YAMLConfigObject(yaml.YAMLObject):
+    """Base class with helper methods to handle configuration YAML data
+
+    This class contains methods to help constructing configuration objects from
+    YAML data.  Then each subclass should implement its standard `to_yaml()`
+    method to be able to dump the whole configuration hierarchy back to YAML.
+    """
+
+    @classmethod
+    def load_from_yaml(cls, config, **kwargs):
+        """Load the YAML configuration
+
+        Load the YAML configuration passed as a *config* data structure with a
+        given *name*.  This method should return an instance of a _YAMLObject
+        subclass.
```

Comment:
Isn't it returning `YAMLConfigObject` which is a subclass of `yaml.YAMLObject`?

Reply:
Yes, that's a mistake in the docstring.

Reply:
Fixed